db/platforms.go

```diff
@@ -130,3 +133,29 @@ func (t *Platforms) Update(ctx context.Context, Options ...dbEngine.BuildSqlOpti
 
 	return t.Table.Update(ctx, Options...)
 }
+
+func GetPlatformFromId(id int32) *PlatformsFields {
+	for _, platform := range platformIds {
+		if platform.Id == id {
+			return &platform
+		}
+	}
+
+	return nil
+}
+
+func GetPlatformsAsSelectedUnits() (res SelectedUnits) {
+	if len(platformIds) == 0 {
+		return nil
+	}
+
+	for _, platform := range platformIds {
```

Comment:
res SelectedUnits must be construct one time

Reply:
didn't get it